pkg/client/devices.go

```diff
@@ -112,14 +112,10 @@ func GetAPIHash(apiSecret string) string {
 	return fmt.Sprintf("%x", sha256.Sum256([]byte(apiSecret)))
 }
 
-// AgentPing is used to receive ping from a device
-type AgentPing struct {
-	AgentCredentials
+// DeviceHeartbeat is used to send heartbeat messages from devices
+type DeviceHeartbeat struct {
 	PingStats
 
-	// Cloud identifier for server (used when running on cloud audio server)
-	CloudID string `json:"cloudId"`
```

Comment:
`CloudID` is used by [this line](https://github.com/jacktriplabs/vs-server/blob/develop/pkg/devices/handlers.go#L527). I think this can be just a `ServerID` and change the implementation in vs-server.

Reply:
Note that I put `CloudID` in the new sibling of this struct: `ServerHeartbeat` in `server.go`. I renamed this one to `DeviceHeartbeat` since it is now unique for devices vs servers.